src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
-                            href="https://onfido.com/terms-of-service/"
-                            style={[styles.textMicro]}
-                        >
-                            {this.props.translate('common.termsOfService')}
-                        </TextLink>
-                    </Text>
-                </Form>
-            </ScreenWrapper>
+            <RequestorOnfidoStep
+                reimbursementAccount={reimbursementAccount}
+                reimbursementAccountDraft={reimbursementAccountDraft}
+                onBackButtonPress={onBackButtonPress}
+                getDefaultStateForField={getDefaultStateForField}
+            />
         );
-    }
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <HeaderWithBackButton
+                title={translate('requestorStep.headerTitle')}
+                stepCounter={STEP_COUNTER}
+                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                onBackButtonPress={onBackButtonPress}
+                shouldShowGetAssistanceButton
+            />
+            <Form
+                formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+                submitButtonText={translate('common.saveAndContinue')}
+                validate={validate}
+                onSubmit={submit}
+                style={STYLES.Form}
+                scrollContextEnabled
+            >
+                <Text>{translate('requestorStep.subtitle')}</Text>
+                <View style={STYLES.LearnMoreContainer}>
+                    <TextLink
+                        style={STYLES.LearnMoreLink}
+                        // eslint-disable-next-line max-len
+                        href="https://community.expensify.com/discussion/6983/faq-why-do-i-need-to-provide-personal-documentation-when-setting-up-updating-my-bank-account"
+                    >
+                        {`${translate('requestorStep.learnMore')}`}
+                    </TextLink>
+                    <Text style={STYLES.LearnMoreSeparator}>{' | '}</Text>
+                    <TextLink
+                        style={STYLES.DataSafeLink}
+                        href="https://community.expensify.com/discussion/5677/deep-dive-security-how-expensify-protects-your-information"
+                    >
+                        {`${translate('requestorStep.isMyDataSafe')}`}
+                    </TextLink>
+                </View>
+                <IdentityForm
+                    translate={translate}
+                    defaultValues={defaultValues}
+                    inputKeys={INPUT_KEYS}
+                    shouldSaveDraft
+                />
+                <CheckboxWithLabel
+                    accessibilityLabel={translate('requestorStep.isControllingOfficer')}
+                    inputID="isControllingOfficer"
+                    defaultValue={getDefaultStateForField('isControllingOfficer', false)}
+                    LabelComponent={LabelComponent}
```

Comment:
```suggestion
                    LabelComponent={renderLabelComponent}
```

Reply:
Hi @rezkiy37, is it a convention inside a project to name such components with `render` prefix? Thanks for all suggestions!

Reply:
Basically, it is a common practise across React/React Native projects to call such render functions like `renderSomething`. Within Expensify code you can see `renderItem`, `renderListEmptyComponent`, `renderSuggestionMenuItem` and so on.

Reply:
Done ✅